### DISCUSSION TITLE
Trying to login 2 Twitter accounts using JWT throws an error for the 2nd account?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [nextauthjs/next-auth#3121] [Help] [answered]
The flow goes like this:

1. `Rock` logs in to Twitter using Chrome, gets JWT stored in the browser, can perform any operation & then logs out.
2. `Trish` logs in to Twitter using the same Chrome, but gets thrown an error saying `Argument username for data.username is missing` for the same code.

I feel like this is expected as JWT probably is stored in the browser & not removed. Am I correct? And will this issue be solved if I use database instead of JWT?

**Comment by deadcoder0904** [comment, marked as answer]
This issue got resolved when I used [database instead of JWT](https://github.com/nextauthjs/next-auth/discussions/3120#discussioncomment-1604188) inside the same browser.

Once I log out, switch my Twitter accounts & login again from my UI then it appropriately shows the correct user. This, however, didn't work with JWT so I believe JWT client might be needed to be deleted or expired when logging out.

**Reply by balazsorban44** [reply]
It is.

https://github.com/nextauthjs/next-auth/blob/ba39efb2566466dfe695bb90513050896d96a2d2/src/core/routes/signout.ts#L40-L45

**Reply by deadcoder0904** [reply]
But it doesn't work when I switch accounts in the same browser. I got an error hence why I switched to using the database. Probably an error with my code then.
